signer/tests/integration/withdrawal_accept.rs

```diff
@@ -243,13 +244,14 @@ async fn accept_withdrawal_validation_happy_path() {
     // Normal: get the signer bitmap for how they voted.
     let bitmap = get_withdrawal_request_signer_votes(&db, &req, &aggregate_key).await;
     // Generate the transaction and corresponding request context.
-    let (accept_withdrawal_tx, ctx) =
+    let (accept_withdrawal_tx, req_ctx) =
         make_withdrawal_accept(&req, sweep_outpoint, aggregate_key, &chain_tip, bitmap);
 
     // This should not return an Err.
-    accept_withdrawal_tx.validate(&db, &ctx).await.unwrap();
+    let ctx = TestSignerContext::from_db(db);
+    accept_withdrawal_tx.validate(&ctx, &req_ctx).await.unwrap();
 
-    testing::storage::drop_db(db).await;
+    testing::storage::drop_db(ctx.into_db()).await;
```

Comment:
`PgStore` is `Clone` so you could probably just have done that instead of `into_db()` (at least right now since both `new_test_db()` and `drop_db()` work explicitly on `PgStore`).

Reply:
I think you need to drop all connections to the database before you can drop it. Maybe that can be forced, I forget 🤔, but that is why I went this route. Let me double check on whether or not you can drop it with existing connections.

Reply:
Oh I already force drop the connections 😅, so this can be simplified.